Userland/Utilities/unzip.cpp

```diff
@@ -191,8 +221,14 @@ int main(int argc, char** argv)
     printf("Archive: %s\n", zip_file_path.characters());
 
     off_t index = 0;
+    StringBuilder target_path;
+    // Currently unzip doesn't support extracting to a specified location. If that is implemented the target_path
+    // should be replaced with the target path from the command line arguements instead of "."
+    target_path.append(realpath(".", nullptr));
+    target_path.append("/");
+    auto target_path_string = target_path.to_string();
```

Comment:
The `target_path` is effectively the same as `cwd` which is defined and unveiled earlier. `target_path` is a more future-proof name and it would make sense to group these together and use the same variable.

This way, if someone adds support for decompressing to a user-specified directory (ie, `unzip -d /output/path`) in the future, they only need to change one line as there's no hard coded assumption of operating in the working directory.